stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/typing/TypingIndicatorView.kt

```diff
@@ -2,29 +2,30 @@ package io.getstream.chat.android.ui.typing
 
 import android.content.Context
 import android.util.AttributeSet
-import android.view.LayoutInflater
 import android.widget.FrameLayout
 import androidx.core.view.isVisible
 import io.getstream.chat.android.client.models.User
 import io.getstream.chat.android.client.models.name
 import io.getstream.chat.android.ui.R
+import io.getstream.chat.android.ui.common.extensions.internal.createStreamThemeWrapper
+import io.getstream.chat.android.ui.common.extensions.internal.streamThemeInflater
 import io.getstream.chat.android.ui.databinding.StreamUiTypingIndicatorViewBinding
 
 public class TypingIndicatorView : FrameLayout {
 
     private val binding: StreamUiTypingIndicatorViewBinding =
-        StreamUiTypingIndicatorViewBinding.inflate(LayoutInflater.from(context), this, true)
+        StreamUiTypingIndicatorViewBinding.inflate(streamThemeInflater, this, true)
```

Comment:
Aren't we getting already wrapped `context` from the base class here? Maybe this will also work:
```
private val binding: StreamUiTypingIndicatorViewBinding =
        StreamUiTypingIndicatorViewBinding.inflate(context, this, true)
```